ReleaseTooling/Sources/FirebaseReleaser/InitializeRelease.swift

```diff
@@ -128,4 +129,9 @@ struct InitializeRelease {
       "'Firebase',[[:space:]]*'\\).*'#\\1\(version)'#\" Podfile"
     Shell.executeCommand(sedCommand2, workingDir: collisionPodfile)
   }
+
+  private static func updateSwiftPackageVersion(path: URL, version: String) {
+    Shell.executeCommand("sed -i.bak -e \"s/\\(let firebaseVersion.*=[[:space:]]*\\).*/\\1" +
```

Comment:
LGTM - if you don't mind can you add a comment above splitting out each section of the regex for future readers and those unfamiliar?